auth/server/src/main/java/com/seagates3/dao/ldap/LdapConnectionManager.java

```diff
@@ -69,12 +69,12 @@ public static void initLdap()
             String msg = "Failed to initialise LDAP.\n" + ex.toString();
             if (retryLdapConnection(ex.getResultCode())) {
               LOGGER.error(msg);
-              IEMUtil.log(
-                  IEMUtil.Level.ERROR, IEMUtil.LDAP_EX,
-                  "An error occurred while establishing ldap " +
-                      "connection. For more information, please refer " +
-                      "Troubleshooting Guide.",
-                  String.format("\"cause\": \"%s\"", ex.getCause()));
+              // IEMUtil.log(
```

Comment:
Why are we commenting instead of removing these. Commented code should be avoided.